lib/middleware/require-github-user.js

```diff
@@ -0,0 +1,32 @@
+const axios = require('axios');
+
+const { GitHubUser } = require('../models');
+const SignedParams = require('../signed-params');
+const getProtocolAndHost = require('../get-protocol-and-host');
+const { SignIn } = require('../messages/flow');
+
+/**
+ * Require the user to have connected their GitHub account
+ */
+module.exports = async function requireGitHubUser(req, res, next) {
+  const slackUser = await res.locals.slackUser.reload({ include: [GitHubUser] });
```

Comment:
Is the `reload necessary`?  Can we just eager load `GitHubUser` wherever we're originally loading the slack user?